vocab/de-de/DaysOfWeek.voc

```diff
@@ -5,3 +5,4 @@ Donnerstag
 Freitag
 Samstag
 Sonntag
+Jedentag
```

Comment:
I don't speak German either - perhaps @gras64 can give us a confirmation on the translation. Otherwise, I'd leave this out (I'd rather not have it than have it be incorrect, since it's harder to change than to add - changing breaks things for people who work around incorrectness)